build/flaws/broken-links.js

```diff
@@ -146,6 +146,14 @@ function getBrokenLinksFlaws(doc, $, { rawContent }, level) {
           `/${VALID_LOCALES.get(homepageLocale.toLowerCase())}/`
         );
       }
+    } else if (hrefNormalized === doc.mdn_url) {
```

Comment:
It seems this will also trigger for "hashtag" links to a specific spot on the same page, like https://developer.mozilla.org/en-US/docs/Web/HTML#references, and those shouldn't be considered broken links. I suspect we should only consider this a broken link if there's no hashtag (i.e., `hrefSplit.length > 1`)?
```suggestion
    } else if ((hrefNormalized === doc.mdn_url) && (hrefSplit.length > 1)) {
```

Reply:
Now, if you have `<a href="/en-US/docs/Web/BrokenLinks#anchor">` (on the `/en-US/docs/Web/BrokenLinks` page) you get a *fixable* flaw that tells you to just change it to `#anchor`.